modules/core/src/lib/init.js

```diff
@@ -43,6 +44,8 @@ if (!global.deck) {
     version,
     log
   };
+
+  initializeShaderModules();
 }
 
 // TODO - Hack, remove when luma.gl 4.1.0-alpha.5 is published
```

Comment:
@ibgreen Is this safe to remove now?

Reply:
Yes, I would assume so, I don't remember this.